PyFlyt/gym_envs/rocket_envs/rocket_base_env.py

```diff
@@ -188,17 +188,14 @@ def begin_reset(
             self.start_orn = rotation
 
         # camera handling
-        if "use_camera" not in drone_options:
-            drone_options["use_camera"] = self.render_mode is not None
-        else:
-            drone_options["use_camera"] |= self.render_mode is not None
+        drone_options["use_camera"] = drone_options.get("use_camera", False) or self.render_mode == "rgb_array"
```

Comment:
Apparently here too it needs to be
```python
drone_options["use_camera"] = drone_options.get("use_camera", False) or bool(self.render_mode)
```

Reply:
Thanks! Was just starting to look there.